contracts/KeyManager.sol

```diff
@@ -7,95 +7,104 @@ contract KeyManager is Ownable {
 
```

Comment:
We should remove `Ownable.sol` in my opinion. We never use owner() or the onlyOwner modifier from Ownable.sol.
We implemented our own solution with adding the owner address to the keys mapping in the constructor.

I would favour an own mapping for addresses together with the use of `Ownable.sol`.
In the verifySignature, we could use this mapping of addresses.

We anyway support only secp256k1 signatures onchain

Reply:
Yea, I forgot to remove it.